components/table/index.jsx

```diff
@@ -148,6 +148,7 @@ class InfiniteTable extends React.Component {
               this.setState({
                 searchTerm: event.target.value,
                 page: 0,
+                isEmpty: false,
```

Comment:
I there really no way to calculate this?

Reply:
Nope table component works like a proxy. It doesn't know about exact data in pages.